packages/protocol/test/governance/validators.ts

```diff
@@ -1297,12 +1306,10 @@ contract('Validators', (accounts: string[]) => {
       })
 
       it("should return that group's member list", async () => {
-        assertAddressesEqual(await validators.getValidators(), [
-          validator1,
-          validator2,
-          validator3,
-          validator4,
-        ])
+        assertAddressesEqual(
+          (await validators.getValidators()).sort(),
```

Comment:
You might be able to use `assert.sameMembers` instead of sorting (but might have to do whatever `assertAddressesEqual` does here, probably standardizing case?).

Or other approach: since we're using `MockRandom` with predictable randomness, instead you could check for the exact ordering we expect.

Reply:
Ditto throughout.